src/styles/blocks/catalog.scss

```
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: transform #{$trans-duration}, color #{$trans-duration};
```

it's not necessary :D
```suggestion
    transition: transform $trans-duration, color $trans-duration;
```